lib/routes/key_data.js

```diff
@@ -74,8 +74,8 @@ module.exports = {
       scopeResults.forEach((keyScope) => {
         response[keyScope.scope] = {
           identifier: keyScope.scope,
-          keyMaterial: ADDITIONAL_KEY_MATERIAL,
-          timestamp: assertionData['fxa-generation']
+          keyRotationSecret: ADDITIONAL_KEY_MATERIAL,
```

Comment:
It's probably worth renaming `ADDITIONAL_KEY_MATERIAL` global for consistency.

Reply:
Good catch!